README.md

```diff
@@ -99,6 +98,45 @@ tuner.run()
 
 The above example runs ASHA with 4 asynchronous workers on a local machine.
 
+## Supported HPO methods
+
+The following hyperparameter optimization (HPO) methods are available in Syne Tune:
+
+Method | Reference | Searcher | Asynchronous? | Multi-fidelity? | Transfer? 
+:--- | :---: | :---: | :---: | :---: | :---: 
+Grid Search | Someone (sometime) | deterministic | no | no | no 
+Random Search | Bergstra, et al. (2011) | random | yes | no | no 
+Bayesian Optimization | Snoek, et al. (2012) | model-based | yes | no | no 
+BORE | Tiao, et al. (2021) | model-based | yes | no | no 
+MedianStoppingRule | Golovin, et al. (2017) | any | yes | yes | no 
+SyncHyperband | Li, et al. (2018) | random | no | yes | no 
+SyncBOHB | Falkner, et al. (2018) | model-based | no | yes | no 
+SyncMOBSTER | Klein, et al. (2020) | model-based | no | yes | no 
+ASHA | Li, et al. (2019) | random | yes | yes | no 
+BOHB | Falkner, et al. (2018) | model-based | yes | yes | no 
+MOBSTER | Klein, et al. (2020) | model-based | yes | yes | no 
+DEHB | Awad, et al. (2021) | random | no | yes | no 
+HyperTune | Li, et al. (2022) | model-based | yes | yes | no 
+PASHA | Bohdal, et al. (2022)| random | yes | yes | no 
+REA | Real, et al. (2019) | model-based | yes | no | no 
+PBT | Jaderberg, et al. (2017) | model-based | no | yes | no 
+ZeroShotTransfer | Wistuba, et al. (2015) | deterministic | yes | no | yes 
+AHSACTS | Salinas, et al. (2021)| random | yes | yes | yes 
+RUSH | Zappella, et al. (2021)| random | yes | yes | yes 
+
+The searchers fall into three broad categories, **deterministic**, **random**, and **model-based**. The random searchers sample candidate hyperparameter configurations uniformly at random, while the model-based searchers sample them non-uniformly at random, according to a model (e.g., Gaussian process, density ration estimator, etc.) and an acquisition function, or by applying an evolutionary algorithm.
+
+Syne Tune also supports [BoTorch](https://github.com/pytorch/botorch) searchers.
+
+## Supported multi-objective optimization methods
+
+Method | Reference | Searcher | Asynchronous? | Multi-fidelity? | Transfer?
+:--- | :---: | :---: | :---: | :---: | :---: 
+Constrained Bayesian Optimization | Gardner, et al. (2014) | model-based | yes | no | no
+MOASHA | Schmucker, et al. (2021) | random | yes | yes | no
```

Comment:
Could we put those in the same table given that they are also optimizers/HPO methods? (with a column multi-objective as we do for transfer for instance)

Reply:
Hmm, maybe a separate table could be a good idea, or at least some kind of structure in the table?
Why not have horizontal lines, and separate like: fifo, multi-fidelity, transfer, multi-objective?

Reply:
And we also have cost-aware BO methods

Reply:
A separate table is deliberate as multi-objective diserves its own in my opinion.  A bunch of method are not included in the baselines. This would enable us to expand the list here too.

Re the formatting: I agree, but couldn't find a way to nicely format tables in markdown. Happy to take another look, but I suggest later, in order not to block the release. What do you think?